settings/settings.go

```diff
@@ -59,13 +59,13 @@ func Load() (*Settings, error) {
 
 	c, err := deis.New(sF.VerifySSL, sF.Controller, sF.Token)
 
-	// Set a custom user agent
-	c.UserAgent = UserAgent
-
 	if err != nil {
 		return nil, err
 	}
 
+	// Set a custom user agent
+	c.UserAgent = UserAgent
```

Comment:
Sorry, that commit accidentally got caught in this PR 😦. I didn't fix it cause the PR it comes from  (#123) was about to land but forgot that it was still caught in endless CI.